client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -363,6 +363,12 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 						homeHtml: sections.map( ( pattern ) => pattern.html ).join( '' ),
 						headerHtml: header?.html,
 						footerHtml: footer?.html,
+						pages: pages
+							.map( ( category ) => pagesMapByCategory[ category ] )
+							.map( ( patterns ) => ( {
+								title: patterns[ 0 ].title,
+								content: patterns[ 0 ].html,
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

I may not have delved into the details, but could you help me understand why we're using `patterns[0]` specifically, instead of `patterns[1]` or `patterns[2]`?

Reply:
> MVP will show only 1 page layout per category

pbxlJb-4EO-p2

😄